src/main/java/thrift/model/Model.java

```diff
@@ -125,16 +131,35 @@
      */
     Transaction getLastTransactionFromThrift();
 
+    /** Returns the current month and year in MMM yyyy format. */
+    String getCurrentMonthYear();
+
+    /** Returns the current month's budget. */
+    double getCurrentMonthBudget();
 
     /** Returns an unmodifiable view of the filtered transaction list */
     ObservableList<Transaction> getFilteredTransactionList();
 
+    /** Filters the view of the transaction list to only show transactions that occur in the current month. */
+    void updateFilteredTransactionListToCurrentMonth();
+
     /**
      * Updates the filter of the filtered transaction list to filter by the given {@code predicate}.
      * @throws NullPointerException if {@code predicate} is null.
```

Comment:
There should be an empty line between description and parameter section

Reply:
Fixed